Gemfile.lock

```diff
@@ -174,6 +174,7 @@ GEM
       rspec-support (~> 3.6.0)
     rspec-support (3.6.0)
     ruby_dep (1.5.0)
+    rubyzip (1.2.4)
```

Comment:
The rubyzip requirement on this line is `rubyzip (1.2.4)` and on L#252 it `rubyzip (~> 1.2.2)`

Reply:
At line 252 we specific to use any version newer or including 1.2 but less 1.3 with the help of this `~>`.
So, `rubyzip (1.2.4)` lies within that range.